src/app-logic/url-handling.js

```diff
@@ -48,6 +48,7 @@ function getDataSourceDirs(
 // "null | void" in the query objects are flags which map to true for null, and false
 // for void. False flags do not show up the URL.
 type BaseQuery = {|
+  v: number,
```

Comment:
I changed the type checking to be a bit more stringent here, and we didn't have the version specified.